nodeadm/internal/kubelet/config.go

```diff
@@ -249,9 +258,18 @@ func (ksc *kubeletConfig) withCloudProvider(cfg *api.NodeConfig, flags map[strin
 
 // When the DefaultReservedResources flag is enabled, override the kubelet
 // config with reserved cgroup values on behalf of the user
-func (ksc *kubeletConfig) withDefaultReservedResources() {
+func (ksc *kubeletConfig) withDefaultReservedResources(cfg *api.NodeConfig) {
 	ksc.SystemReservedCgroup = ptr.String("/system")
 	ksc.KubeReservedCgroup = ptr.String("/runtime")
+	maxPods, ok := getMaxPod(cfg.Status.Instance.Type)
+	if !ok {
+		return
+	}
+	ksc.KubeReserved = map[string]string{
+		"cpu":               strconv.Itoa(getCPUMillicoresToReserve()),
+		"ephemeral-storage": "1Gi",
+		"memory":            strconv.Itoa(getMemoryMebibytesToReserve(maxPods)),
```

Comment:
can you format string these to have the units? (`Mi`, `m`)
https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh#L522